package.json

```diff
@@ -20,66 +21,83 @@
-    "@angular/animations": "12.2.8",
-    "@angular/cdk": "12.2.8",
-    "@angular/common": "12.2.8",
-    "@angular/compiler": "12.2.8",
-    "@angular/core": "12.2.8",
-    "@angular/fire": "^7.0.4",
-    "@angular/forms": "12.2.8",
-    "@angular/material": "12.2.8",
-    "@angular/platform-browser": "12.2.8",
-    "@angular/platform-browser-dynamic": "12.2.8",
-    "@angular/upgrade": "12.2.8",
+    "@angular/animations": "15.1.1",
+    "@angular/cdk": "15.1.1",
+    "@angular/common": "15.1.1",
+    "@angular/compiler": "15.1.1",
+    "@angular/core": "15.1.1",
+    "@angular/fire": "^7.5.0",
+    "@angular/forms": "15.1.1",
+    "@angular/material": "15.1.1",
+    "@angular/platform-browser": "15.1.1",
+    "@angular/platform-browser-dynamic": "15.1.1",
+    "@angular/upgrade": "15.1.1",
     "@schuchard/prettier": "^5.1.0",
-    "@types/angular": "^1.8.2",
-    "@types/angular-material": "^1.1.73",
     "angular": "npm:@xlts.dev/angular@1.9.1",
     "angular-animate": "npm:@xlts.dev/angular-animate@^1.9.1",
     "angular-aria": "npm:@xlts.dev/angular-aria@^1.9.1",
     "angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
     "angular-messages": "npm:@xlts.dev/angular-messages@^1.9.1",
     "angular-sanitize": "npm:@xlts.dev/angular-sanitize@^1.9.1",
-    "firebase": "^9.0.0",
-    "rxfire": "^6.0.0",
-    "rxjs": "^6.6.7",
-    "tslib": "^2.3.1",
-    "zone.js": "~0.11.4"
+    "firebase": "^9.16.0",
+    "ngx-toastr": "^16.0.2",
+    "rxfire": "^6.0.3",
+    "rxjs": "^7.8.0",
+    "tslib": "^2.4.1",
+    "zone.js": "~0.12.0"
   },
   "devDependencies": {
-    "@angular-devkit/architect": "^0.1202.7",
-    "@angular-devkit/build-angular": "12.2.8",
-    "@angular/cli": "12.2.8",
-    "@angular/compiler-cli": "12.2.8",
-    "@angular/language-service": "12.2.8",
+    "@angular-devkit/architect": "^0.1501.2",
+    "@angular-devkit/build-angular": "~15.1.1",
+    "@angular-eslint/builder": "15.2.0",
+    "@angular-eslint/eslint-plugin": "15.2.0",
+    "@angular-eslint/eslint-plugin-template": "15.2.0",
+    "@angular-eslint/schematics": "15.2.0",
+    "@angular-eslint/template-parser": "15.2.0",
+    "@angular/cli": "15.1.1",
+    "@angular/compiler-cli": "15.1.1",
+    "@angular/language-service": "15.1.1",
     "@playwright/test": "^1.30.0",
-    "@types/gulp": "^4.0.9",
-    "@types/node": "^14.17.20",
+    "@types/angular": "^1.8.4",
+    "@types/angular-animate": "^1.5.11",
+    "@types/angular-aria": "^1.7.2",
+    "@types/angular-material": "^1.1.73",
+    "@types/angular-sanitize": "^1.8.1",
+    "@types/gulp": "^4.0.10",
+    "@types/node": "^18.11.18",
+    "@typescript-eslint/eslint-plugin": "5.49.0",
+    "@typescript-eslint/parser": "5.49.0",
     "codelyzer": "^6.0.2",
     "debug": "^4.3.4",
-    "firebase-tools": "^9.19.0",
+    "eslint": "~8.32.0",
+    "eslint-config-prettier": "8.6.0",
+    "eslint-import-resolver-typescript": "^2.5.0",
+    "eslint-plugin-ban": "^1.6.0",
+    "eslint-plugin-import": "^2.27.5",
```

Comment:
Is this needed?

Reply:
I do not know why, but the previous time I had an error lint:

<img width="906" alt="image" src="https://user-images.githubusercontent.com/5073455/230589438-9a024cf6-ac9b-4674-8087-018e964aa9ea.png">

I see that it has disappeared. I have removed the dependency